west.yml

```diff
@@ -59,7 +59,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 8182ebc2d257c309b9c50fc0d895a4b88a672a1b
+      revision: pull/1162/head
```

Comment:
Let's do the final manifest update here (`dfd1fad51fa591b44ed44159e54a97625d317a99`).

Just need to wait for https://github.com/nrfconnect/sdk-nrf/pull/11109 as it'll trigger conflict.